iothub/service/src/AmqpServiceClient.cs

```diff
@@ -324,21 +313,9 @@ public override Task<DeviceStreamResponse> CreateStreamAsync(string deviceId, De
         /// <param name="deviceId">Device Id</param>
         /// <param name="moduleId">Module ID</param>
         /// <param name="deviceStreamRequest">Configuration needed for initiating a cloud-to-device stream.</param>
-        /// <returns>The result of the cloud-to-device stream request, or null of the request itself could not be completed.</returns>
-        public override Task<DeviceStreamResponse> CreateStreamAsync(string deviceId, string moduleId, DeviceStreamRequest deviceStreamRequest)
-        {
```

Comment:
Why wouldn't module-id be needed anymore?

Reply:
and the API without cancellation token?

Reply:
A method with module param still exists.

One of the bugs was to eliminate overloads without cancellation tokens. The ones with it just have a default parameter for it, which is how we've been handling this elsewhere (like ADT).

Reply:
It still is. This used to be 4 APIs, where 2 needed cancellation tokens, and the other 2 didn't. Git diff just makes it confusing to see that